docs/tutorial/setup.rst

```diff
@@ -24,6 +24,8 @@ Setup
 .. _Matplotlib: https://matplotlib.org
 .. _Pysam: https://pysam.readthedocs.io
 .. _Bioconda: https://bioconda.github.io
+.. _WSL: https://docs.microsoft.com/en-us/windows/wsl/about
+.. _WSL Documentation: https://docs.microsoft.com/en-us/windows/wsl/install-win10
```

Comment:
Isn't WSL2 much faster, and the one that Docker Desktop for Windows has nice integration with?

WSL2 is available for all Windows 10 users, even the Home edition

Reply:
I used WSL as a generic acronym here; I don't use WSL/WSL2, if the distinction is important (i.e. WSL2 is an entirely separate concept) then I will specify WSL2 in the text. The instructions I linked there cover WSL2 as far as I understand and I used them to setup WSL2 in my test.